src/user.rs

```diff
@@ -0,0 +1,177 @@
+//! Tasks defined rust_actix-basic.pdf
+//! Using endpoints at: https://jsonplaceholder.typicode.com/<API>
+
+use validator::Validate;
+use validator_derive;
+
+use reqwest;
+use serde::{Deserialize, Serialize};
+
+/// Submits a GET request to a specified URL, and returns a vector of users. Expected response is in json format, and can be parsed by the `User` struct with accessible elements by key name.
+pub async fn get_users(url: &str) -> Result<Vec<User>, Box<dyn std::error::Error>> {
```

Comment:
What type of error will the end user get if there is an issue connecting to the API?